cityofaustin/atd-moped
# Make Moped <> Data Tracker sync compatible with new components schema

## Associated issues
- https://github.com/cityofaustin/atd-data-tech/issues/11216

## Testing
**URL to test:** local

**Steps to test:**
1. To test locally, you'll need to uncomment [L122](https://github.com/cityofaustin/atd-moped/blob/6213fdb27f4808da26a9e9147ba4447355535313/moped-editor/src/views/projects/projectView/ProjectSummary/ProjectSummaryKnackDataTrackerSync.js#L122) :(
2. Create a project with a signal component
3. On the project's summary tab, click the "Synchronize with Data Tracker" link to push this moped project to the Data Tracker test environment. 
4. Keep an eye on the top right-hand corner of the page to see the green "Success" toast 🥪 
5. Now move your eyes back to where you clicked the "Synchronize with Data Tracker" link, and you should instead see a hyperlinked signal ID. Clicking that ID will take you to a knack app.
6. If you'd like to be thorough, you can login to the test Knack app [here](https://builder.knack.com/atd/test-25-feb-2022-dt-austin-transportation-data-tracker/records/objects/object_201?builder_table_sort=field_3858|desc) using the admin builder creds in our password store. You should see your project in the first row.

---
#### Ship list
